web/jenkins.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Jenkins slave
+
+[Service]
+User=omero
+WorkingDirectory=/tmp
+PIDFile=/tmp/jenkins.pid
+Type=forking
+Restart=on-failure
+RestartSec=10
+ExecStart=/tmp/jenkins-slave.sh
```

Comment:
Does this start jenkins as a background (daemon) process, or in the foreground? If it's in the foreground you can use Type `simple` instead and omit the `ExecStop`, e.g. see https://github.com/ome/omero-install/blob/develop/linux/omero-systemd.service